support/troubleshooting/troubleshooting-operating-system-issues.adoc

```diff
@@ -10,3 +10,46 @@ toc::[]
 
 // Investigating kernel crashes
 include::modules/investigating-kernel-crashes.adoc[leveloffset=+1]
+
+include::modules/troubleshooting-enabling-kdump.adoc[leveloffset=+2]
+
+include::modules/troubleshooting-enabling-kdump-day-one.adoc[leveloffset=+2]
+
+== Testing the kdump configuration
```

Comment:
Add unique IDs here and a few more headings below (also consider ID for the "Additional resources" like you had in the 4.10 PR).